examples/misc/pubspec.yaml

```diff
@@ -2,7 +2,7 @@ name: examples
 description: dart.dev example code.
 
 environment:
-  sdk: ">=2.17.0-0 <3.0.0"
+  sdk: ">=2.17.0 <3.0.0"
```

Comment:
This will break beta CI until beta gets a new version in a couple of weeks. I'd suggest reverting